.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
I think you can delete this file from the git repository as it is not needed for the project.

Add the `.vscode` folder to the `.gitignore` file and then delete it from the repository